### PR TITLE
Add solution for 6.ICE.1

### PR DESCRIPTION
**Pull request by raivatshah**

### CODE REVIEW

File: M6/6.ICE.1/dist/main.js [hunk 1]
```diff
@@ -0,0 +1,1480 @@
+/*
```
Comment (jiachen247) [inline M6/6.ICE.1/dist/main.js]: Let's not check in compiled files

File: M6/6.ICE.1/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// webpack.config.js
+const path = require('path');
+
+module.exports = {
+  mode: "development" , // or "production"
+  entry: "./src/index.js",
+//   output: {
```
Comment (jiachen247) [inline M6/6.ICE.1/webpack.config.js]: why is this commented out? can we use the default

```
  output: {
    filename: "main.js",
    path: path.resolve(__dirname, "dist")
  }
```

**Review by jiachen247 — CHANGES_REQUESTED** [review]

File: M6/6.ICE.1/webpack.config.js (hunk 2 above)
Comment (raivatshah) [reply M6/6.ICE.1/webpack.config.js]: done!

File: M6/6.ICE.1/dist/main.js (hunk 1 above)
Comment (raivatshah) [reply M6/6.ICE.1/dist/main.js]: Ok!

**Review by jiachen247 — APPROVED** [review]
LGTM

**Pull request merged**